src/ArchiveAdmin.php

```diff
@@ -313,6 +313,7 @@ public function Breadcrumbs($unlinked = false)
      */
     public function getManagedModels()
     {
+        $archivedModels = [];
         $models = $this->getVersionedModels();
```

Comment:
Looking at the logic further down, I think this is what it was meant to be before.
```suggestion
        $archivedModels = $this->getVersionedModels();
```